File: app/src/main/java/com/app/cheffyuser/home/fragments/KitchenFragment.java
package com.app.cheffyuser.home.fragments;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;
import androidx.viewpager.widget.ViewPager;

import com.app.cheffyuser.R;
import com.app.cheffyuser.home.adapter.Sub2TabsAdapter;
import com.app.cheffyuser.home.adapter.ViewPagerAdapter_for_internal_tab;
import com.google.android.material.tabs.TabLayout;

/**
 * A simple {@link Fragment} subclass.
 */
public class KitchenFragment extends Fragment {


    public KitchenFragment() {
        // Required empty public constructor
    }



    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view= inflater.inflate(R.layout.fragment_kitchen, container, false);

        TabLayout tabLayout = (TabLayout)view. findViewById(R.id.details_review_tab_layout);
        tabLayout.addTab(tabLayout.newTab().setText("Details"));
        tabLayout.addTab(tabLayout.newTab().setText("Review"));
        tabLayout.setTabGravity(TabLayout.GRAVITY_FILL);




        final ViewPager viewPager =(ViewPager)view.findViewById(R.id.details_review_viewpager);
        Sub2TabsAdapter tabsAdapter = new Sub2TabsAdapter(getChildFragmentManager(), tabLayout.getTabCount());
        viewPager.setAdapter(tabsAdapter);
        viewPager.addOnPageChangeListener(new TabLayout.TabLayoutOnPageChangeListener(tabLayout));
        tabLayout.setOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
            @Override
            public void onTabSelected(TabLayout.Tab tab) {
                viewPager.setCurrentItem(tab.getPosition());
            }
            @Override
            public void onTabUnselected(TabLayout.Tab tab) {

            }
            @Override
            public void onTabReselected(TabLayout.Tab tab) {

            }
        });



        return view;
    }


}

File: app/src/main/java/com/app/cheffyuser/home/fragments/KitchenFragment.kt
package com.app.cheffyuser.home.fragments


import androidx.fragment.app.Fragment


/**
 * A simple [Fragment] subclass.
 *
 * Check implementation in  [PlateFragment]
 */
class KitchenFragment : PlateFragment() {


 /*   override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment

        return inflater.inflate(R.layout.fragment_kitchen, container, false)
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)


        initTablayout()
    }


    private fun initTablayout() {
        tab_layout.tabGravity = TabLayout.GRAVITY_FILL

        val adapter = ReviewDetailbarAdapter(childFragmentManager)
        adapter.addFragments(PlateDetailsFragment())
        adapter.addFragments(PlateReviewFragment())

        viewpager.adapter = adapter
        viewpager.offscreenPageLimit = 2

        tab_layout.setupWithViewPager(viewpager)
    }*/

}
